partner_communication_revision/wizards/revision_preview.py

```diff
@@ -75,32 +70,34 @@ def preview(self):
         if not self.revision_id or not self.res_id:
             return self._reload()
         record = self.env[self.model].browse(int(self.res_id))
-        partner_id = record.id if self.model == 'res.partner' else \
+        partner_id = \
+            record.id if self.model == 'res.partner' else \
             record.partner_id.id
         config = self.revision_id.config_id
         job_vals = {
             'config_id': config.id,
-            'object_ids': self.res_id,
+            'object_ids': self.res_id.id,
             'partner_id': partner_id,
```

Comment:
move selection field to reference field.
selection was filled with ids
reference is like a referencial field so have to take the id